### DISCUSSION TITLE
not getting the 'lib' folder in my foundry-f23

### DISCUSSION THREAD

**Discussion by SairajMN** [Cyfrin/foundry-full-course-cu#2017] [Q&A] [answered]
![Screenshot 2024-07-14 201126](https://github.com/user-attachments/assets/3118bfa4-cb45-4d06-8883-301873c11626)

**Comment by SairajMN** [comment]
after all setup i should get all files and folders right, i am i doing it wrong or what?
like this one 
![Screenshot 2024-07-14 201858](https://github.com/user-attachments/assets/54a3b40b-7bd2-467a-88f3-807c99d85b48)
patrick has lib folder.
I did the exact same way that shown in video and along the video 

can anyone tell me what's wrong happend?

**Reply by EngrPips** [reply]
You have a `lib` folder as well.

**Reply by SairajMN** [reply]
it's not mine, it's SS of patrick folders.

**Reply by SairajMN** [reply, marked as answer]
i now i got it, deleted the whole foundry things and redo the thing again and now i got it in my folder 

![Screenshot 2024-07-14 205332](https://github.com/user-attachments/assets/760423ea-844b-4553-88b5-2d4c11e45536)


thank you for ur time i appreciate it.

**Reply by EngrPips** [reply]
You are welcome. Keep crushing it.
